src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -0,0 +1,280 @@
+{#**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends 'PrestaShopBundle:Admin:layout.html.twig' %}
+
+{% block content %}
+{% if notHostMode %}
+    <script type="text/javascript">
```

Comment:
FYI, no need to specify `type="text/javascript"` on HTML5 files 🙂

Reply:
I know, it's mostly a "copy/paste/migration" template :)

Reply:
done btw